File: python/day3apr6/assign/1.py
"""
1.Write a menu driven code to store employee information in dictionary(Only one employee details)
->Add employee details
->delete emplpoyee
->Display employee
Take input from user
Properties: Name,Age,Gender,Place,Salary,Previous_company
"""

emp = []
# menu
def menu():
	print("""
		press[1] : add
		press[2] : delete
		press[3] : display
		press[4] : exit
""")
# adding members
def add():
	temp = {}
	temp["name"] = input("name : ")
	temp["age"] = input("age : ")
	temp["gender"] = input("gender : ")
	temp["place"] = input("place : ")
	temp["salary"] = input("salary : ")
	temp["previous_company"] = input("previous_company : ")

	emp.append(temp)

# deleting members
def delete():
	ch = (input("name to delete :"))
	for i in emp:
		if i["name"] == ch:
			emp.remove(i)

# display
def display():
	print(emp)

while 1:
	menu()
	but = int(input("Choice : "))
	if but == 1:
		add()
	elif but == 2:
		delete()
	elif but == 3:
		display()
	elif but == 4:
		break
	else:
		print("Invalid")

File: c_assignments/assignment24marday13/1.c
#include<stdio.h>

//Q1.Create a Employee structure to store the EMPID,NAME,Phone,Email,Salary.(Use array of structure)

int main()
{
	struct employee
	{
		int empID;
		char name[100];
		char phone[10];
		char email[20];
		int salary;
	};

	typedef struct employee emp;
	emp em[10];

	// Taking input from user
	for(int i=0; i<2; i++)
	{
		printf("Enter empID of e%d : ", i+1);
		scanf("%d", &em[i].empID);
	
		printf("Enter name of e%d : ", i+1);
		scanf("%s", em[i].name);

		printf("Enter phone of e%d : ", i+1);
		scanf("%s", em[i].phone);

		printf("Enter email of e%d : ", i+1);
		scanf("%s", em[i].email);

		printf("Enter salary of e%d : ", i+1);
		scanf("%d", &em[i].salary);
	}

	// Printing Employee details
	for(int i=0; i<2; i++)
	{
		printf("e%d : empID | name | phone | email | salary\n", i);
		printf("e%d : %d | %s | %s | %s | %d \n", i+1, em[i].empID, em[i].name, em[i].phone, em[i].email, em[i].salary);
	}

	return 0;
}
